packages/network-controller/src/create-auto-managed-network-client.test.ts

```diff
@@ -18,7 +18,7 @@ describe('createAutoManagedNetworkClient', () => {
     {
       type: NetworkClientType.Custom,
       rpcUrl: 'https://test.chain',
-      chainId: '0x1337',
+      caipChainId: 'eip155:1337',
```

Comment:
should this be the decimal form of 0x1337? (4919)

Reply:
oh wait, this one actually was already in base 10, but also 0x prefixed. the hex form should have been 0x539. RIP